test/libgit2.jl

```diff
@@ -346,6 +348,7 @@ mktempdir() do dir
                 LibGit2.add!(repo, test_file)
                 status = LibGit2.GitStatus(repo)
                 @test length(status) != 0
+                @test status[0] == nothing
```

Comment:
I don't know how much it matters but I remember seeing Tony Kelman say somewhere that it's better to use `===` for `nothing`.